doc/host-applications.rst

```diff
@@ -128,8 +125,8 @@ We can now use the standard C++ compiler to build all this into an
 executable, leveraging ``spicy-config`` to add the necessary flags
 for finding includes and libraries::
 
-    # clang++ -o my-http my-http-host.cc my-http.cc my-http-linker.cc $(spicy-config --cxxflags --ldflags)
-    # ./my-http $'GET index.html HTTP/1.0\n'
+    # clang++ -o my_http my_http-host.cc my_http___linker__.cc my_http_MyHTTP.cc $(spicy-config --cxxflags --ldflags)
```

Comment:
Yeah, we were inconsistent and used this at a couple places. I decided against because I think it makes it harder to read/understand what's going on when looking at the docs, as one needs to know/anticipate what `spicy-config --cxx` and keep that in mind. But let me add a note pointing out the alternative.